{"uid":"e190b8822fcfab23","name":"test_refresh_token_success[authorize_admin]","fullName":"tests.auth.test_04_refresh#test_refresh_token_success","historyId":"5500bb5a5d5b2b50f7748091669c9dab","time":{"start":1744589586340,"stop":1744589587111,"duration":771},"description":"Тест успешного обновления токена через refresh-токен.","descriptionHtml":"<p>Тест успешного обновления токена через refresh-токен.</p>\n","status":"skipped","statusMessage":"XFAIL https://ru.yougile.com/team/01c2a92f1328/#IT-468\n\nAssertionError: Ожидался статус 200, но получен 500","statusTrace":"api_client = <api_tests.utils.api_client.APIClient object at 0x7f5d0653ef90>\nauthorize_func = <function authorize_admin at 0x7f5d06616520>\n\n    @pytest.mark.smoke\n    @pytest.mark.xfail(reason=\"https://ru.yougile.com/team/01c2a92f1328/#IT-432\")\n    @pytest.mark.xfail(reason=\"https://ru.yougile.com/team/01c2a92f1328/#IT-468\")\n    @pytest.mark.parametrize(\"authorize_func\", [authorize_client, authorize_partner, authorize_admin])\n    def test_refresh_token_success(api_client, authorize_func):\n        \"\"\"Тест успешного обновления токена через refresh-токен.\"\"\"\n    \n        auth_response = authorize_func(api_client)\n        access_token = auth_response[\"access_token\"][\"token\"]\n        refresh_token = auth_response[\"refresh_token\"][\"token\"]\n    \n        # Передаем refresh-токен в заголовке Authorization\n        headers = {\"Authorization\": f\"Bearer {refresh_token}\"}\n        response = api_client.post(\"/auth/refresh\", headers=headers)\n    \n>       assert_response_status_OK(api_client.last_response.status_code, 200)\n\napi_tests/tests/auth/test_04_refresh.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nactual_status = 500, expected_status = 200\n\n    def assert_response_status_OK(actual_status, expected_status):\n>       assert actual_status == expected_status, f\"Ожидался статус {expected_status}, но получен {actual_status}\"\nE       AssertionError: Ожидался статус 200, но получен 500\n\napi_tests/utils/asserts.py:33: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"api_client","time":{"start":1744589584574,"stop":1744589584574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"api_client::0","time":{"start":1744589587455,"stop":1744589587455,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='https://ru.yougile.com/team/01c2a92f1328/#IT-468')"},{"name":"tag","value":"smoke"},{"name":"parentSuite","value":"tests.auth"},{"name":"suite","value":"test_04_refresh"},{"name":"host","value":"fv-az1477-977"},{"name":"thread","value":"1852-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.auth.test_04_refresh"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"authorize_func","value":"<function authorize_admin at 0x7f5d06616520>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["smoke","@pytest.mark.xfail(reason='https://ru.yougile.com/team/01c2a92f1328/#IT-468')"]},"source":"e190b8822fcfab23.json","parameterValues":["<function authorize_admin at 0x7f5d06616520>"]}